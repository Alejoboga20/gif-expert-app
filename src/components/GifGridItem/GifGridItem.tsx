import { Card } from '@nextui-org/react';

const GifGridItem = ({ title, url }: GifGridItemProps) => {
	return (
		<Card bordered hoverable shadow>
			<Card.Body>
				<Card.Image
					src={url}
					alt={title}
					objectFit='cover'
					width='100%'
					height='10rem'
				></Card.Image>
			</Card.Body>
			<Card.Footer>
				<p>{title}</p>
			</Card.Footer>
		</Card>
	);
};

interface GifGridItemProps {
	title: string;
	url: string;
}

export default GifGridItem;
