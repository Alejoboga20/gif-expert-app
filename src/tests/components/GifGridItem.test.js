import React from 'react';
import '@testing-library/jest-dom';
import { shallow } from 'enzyme';
import GifGridItem from '../../components/GifGridItem';

describe('GifGridItem Tests', () => {
  let wrappper = shallow(<GifGridItem />);

  test('should Display GifGridItem properly', () => {
    expect(wrappper).toMatchSnapshot();
  });
});
