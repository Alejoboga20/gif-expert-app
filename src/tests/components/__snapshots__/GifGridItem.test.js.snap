// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GifGridItem Tests should Display GifGridItem properly 1`] = `
<div
  className="card animate__animated animate__fadeIn"
>
  <img />
  <p />
</div>
`;
